app/components/provider_interface/application_feedback_component.rb

```diff
@@ -0,0 +1,53 @@
+module ProviderInterface
+  class ApplicationFeedbackComponent < ViewComponent::Base
+    include ViewHelper
+    include StatusBoxComponents::CourseRows
+
+    attr_reader :application_choice
+
+    def initialize(application_choice:, options: {})
+      @application_choice = application_choice
+      @options = options
+    end
+
+    def render?
+      application_choice.rejected?
+    end
+
+    def rejected_rows
+      if application_choice.rejected_by_default
+        rows = [
+          {
+            key: 'Automatically rejected',
+            value: application_choice.rejected_at.to_s(:govuk_date),
+          },
+        ]
+        if application_choice.reject_by_default_feedback_sent_at.present?
+          rows + [
+            {
+              key: 'Feedback sent',
```

Comment:
Not very sure around if this works with the current established component content pattern: I noticed that even when feedback is updated it will always appear as "Feedback sent", if it's possible in distinguishing between feedback being sent for the first time vs feedback being updated, maybe we can surface those differences in those components as "Feedback updated"? Let me know your thoughts

Reply:
Discussed in person but for posterity: we shouldn't be able to update feedback, which is now possible only through going back in the browser. A few options around mitigating this are to redirect back to the feedback tab with a flash message saying this has already been done